app/controllers/slides_controller.rb

```diff
@@ -52,14 +51,14 @@ def show
 
   def new
     @slide = Slide.new
+    render "slides/#{CloudConfig::service_name}/new"
```

Comment:
__[rubocop]__
* Style/ColonMethodCall: Do not use `::` for method calls. [:ref](https://github.com/bbatsov/ruby-style-guide#double-colons)